samples/bluetooth/mesh/shell_client/README.rst

```diff
@@ -0,0 +1,193 @@
+.. _bluetooth_mesh_shell_client:
```

Comment:
In general I think this documentation looks fine. Since the capabillities of a client that is hooked up to mesh shell exeeds the mentioned usecases that is described here, I'm not 100% sure what will be the final result of this sample when it is going to be merged to NCS. But it will be close to what you have in your current version. I just want you to keep that in mind before spending too much time ironing out the last details of the docs here.